package.json

```diff
@@ -48,6 +48,7 @@
     "next-mdx-remote": "^4.4.1",
     "next-query-params": "^5.0.0",
     "next-seo": "^6.5.0",
+    "nuqs": "^1.19.3",
```

Comment:
can we remove the other package now?

Reply:
Yes! I was planning to handle this migration in the 2nd PR (one of the reasons I wanted to separate out the work actually.